api/templates/api.mustache

```diff
@@ -71,6 +95,22 @@ func (r {{#structPrefix}}{{&classname}}{{/structPrefix}}Api{{operationId}}Reques
 	return r.ApiService.{{nickname}}Execute(r)
 }
 
+// Sets additional descriptive text in the error message if this specific
+// request fails, indicating that it is intended to be used only on OSS
+// servers.
+func (r {{#structPrefix}}{{&classname}}{{/structPrefix}}Api{{operationId}}Request) OnlyOSS() {{#structPrefix}}{{&classname}}{{/structPrefix}}Api{{operationId}}Request {
+	r.isOnlyOSS = true
+	return r
+}
+
+// Sets additional descriptive text in the error message if this specific
+// request fails, indicating that it is intended to be used only on cloud
+// servers.
+func (r {{#structPrefix}}{{&classname}}{{/structPrefix}}Api{{operationId}}Request) OnlyCloud() {{#structPrefix}}{{&classname}}{{/structPrefix}}Api{{operationId}}Request {
+	r.isOnlyCloud = true
+	return r
+}
```

Comment:
I'd vote to keep things simple for now and have the OSS-vs-Cloud toggle at the API level, and not per-request

Reply:
Done! Should make it easier to keep track of what has been flagged for a specific server type for sure.